lib/screens/container/list.dart

```diff
@@ -1,13 +1,15 @@
 import 'package:flutter/material.dart';
-
+import 'package:percent_indicator/circular_percent_indicator.dart';
 import 'package:shipanther/bloc/container/container_bloc.dart';
 import 'package:shipanther/l10n/shipanther_localization.dart';
 import 'package:shipanther/screens/container/add_edit.dart';
 import 'package:shipanther/extensions/container_extension.dart';
+import 'package:shipanther/widgets/filter_button.dart';
 import 'package:shipanther/widgets/shipanther_scaffold.dart';
 import 'package:trober_sdk/api.dart' as api;
+import 'package:flutter_bloc/flutter_bloc.dart';
 
-class ContainerList extends StatelessWidget {
+class ContainerList extends StatefulWidget {
```

Comment:
Why do we need to convert this to stateful? I don't see any state that is needed to be set.